main.go

```diff
@@ -147,6 +147,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Defaults for service operators
+
+	// Acquire environmental defaults and initialize Glance and GlanceAPI defaults with them
+	glanceDefaults := glancev1.GlanceDefaults{
```

Comment:
this could get messy down the road. may want to consider putting it in a function down the road, but lets go with it for now